### ISSUE TITLE
partner logos section missing

### ISSUE THREAD

**Issue by figart** [digett-trs/trs#50]
The section on the home page featured partner logos that move horizontally across the page is missing.

**Comment by figart** [comment]
This may be something we gain from merging back with main branch? Or???

**Comment by emontes** [comment]
> This may be something we gain from merging back with main branch? Or???
Let me double check, I think I've seen the code for that part in one of my many walkthroughs of the general code.

**Comment by emontes** [comment]
Ok the problem is in the file layoutSearch.js here
![image](https://user-images.githubusercontent.com/1318017/198847709-e4d7c1e7-2d1f-4a80-a11b-be9f290094ab.png)

we are displaying rawEntities[1] and [2]. Previous the fix we was displaying rawEntities[1] and [0]

Ok this entities comes from Sanity
![image](https://user-images.githubusercontent.com/1318017/198847788-ed775e51-9a3b-4659-8535-a4bbdbf8d5bd.png)

In some point someone has added extra homeCarousel and moved the order of "Our partners" (that was zero).

**Comment by figart** [comment]
I saw a partners section on the home page before I switched the build configuration in Gatsby Cloud to use the main branch. However, it should go below the property listings, not above.

**Comment by figart** [comment]
@emontes I'm not seeing the partners section at https://trs.gatsbyjs.io/. Am I looking in the wrong place?

**Comment by emontes** [comment]
> @emontes I'm not seeing the partners section at https://trs.gatsbyjs.io/. Am I looking in the wrong place?

Sorry, my mistake, I was doing the commits to the development branch. And now the one that are deploying to https://trs.gatsbyjs.io/ is the main branch.

I already did the necessary merge and you can see the latest changes.

**Comment by figart** [comment]
Okay, looks like we got this working again, only it's going way too fast. I'm not sure how the speed is measured, but if we could make it go about half the speed it's going currently I think it would be just about right.

**Comment by emontes** [comment]
The speed here is measured in seconds in the variable --animationspeed, it was in 30 seconds and I've changed it to 50 seconds.

You can see it in https://build-ffbe911b-4f3c-402b-96eb-128d3b8f7663.gtsb.io/

**Comment by figart** [comment]
Yes, this looks great, thank you! I'm super curious, though... how did you create this build?

**Comment by emontes** [comment]
![image](https://user-images.githubusercontent.com/1318017/199318216-25f02ce0-e989-4e3b-a0bc-049322ffccf5.png)

If I click in the green checkmark of the "slow down the speed...." commit it shows me Gatsby build service and there I clicked on "Details"

**Comment by figart** [comment]
Oh my gosh!! That's so cool! I think you may be the first person on my team to discover that, @emontes!! I love it! Thank you!

**Comment by emontes** [comment]
Do you know the Legend of the Donkey that played the Flute? Hehe, that's what happened to me here.

**Comment by figart** [comment]
Lol... too funny. Does that mean you're an ass? 😂

**Comment by emontes** [comment]
jaja, The same thing happened to me as the donkey, it was a mere coincidence

**Comment by figart** [comment]
simply a case of "good things happen to good people"
